contracts/mocks/SafeERC20Helper.sol

```diff
@@ -54,8 +54,8 @@ contract SafeERC20Helper {
     IERC20 private _succeeding;
 
     constructor () public {
-        _failing = IERC20(new ERC20FailingMock());
-        _succeeding = IERC20(new ERC20SucceedingMock());
+        _failing = IERC20(address(new ERC20FailingMock()));
```

Comment:
This is actually because `ERC20FailingMock` doesn't extend `IERC20`. If it did it would just automatically cast to `IERC20`. This should be improved in `master`...

Reply:
Yup, but to do so we'd need to implement some more methods (e.g. `balanceOf`), which had some issues of its own that I can't recall right now.